packages/devtools_app/test/flutter/inspector_screen_test.dart

```diff
@@ -168,5 +170,9 @@ void main() {
       // TODO(jacobr): also verify that the service extension buttons look
       // visually disabled.
     });
+
+    // TODO(jacobr): add screenshot tests that connect to a test application
```

Comment:
How would you feel about using pre-captured VM snapshots for this one?  That will save us the trouble of bootstrapping an integration test environment and these screenshot tests can be run as simple unit tests.

Reply:
at some point the inspector will be refactored enough to support running offline based on json snapshots describing the app. Until that is done, this is not as simple as you might expect.

Reply:
you need a full flutter system and there is a lot of chat back on forth over the widget_inspector api to render.